Content.Server/Objectives/Systems/KillPersonConditionSystem.cs

```diff
@@ -86,11 +85,10 @@ private void OnHeadAssigned(EntityUid uid, PickRandomHeadComponent comp, ref Obj
         }
 
         var allHeads = new List<EntityUid>();
-        foreach (var mind in allHumans)
+        foreach (var minds in allHumans)
         {
-            // RequireAdminNotify used as a cheap way to check for command department
-            if (_job.MindTryGetJob(mind, out var prototype) && prototype.RequireAdminNotify)
-                allHeads.Add(mind);
+            if (TryComp<MindComponent>(minds, out var mind) && mind.OwnedEntity != null && HasComp<CommandStaffComponent>(mind.OwnedEntity.Value))
```

Comment:
Same as @deltanedas's suggesto for the other similar case.
```suggestion
            if (TryComp<MindComponent>(minds, out var mind) && mind.OwnedEntity is { } ent && HasComp<CommandStaffComponent>(ent))
```

Since I'm also new to C#, [here's an explanation](https://stackoverflow.com/questions/60417114/what-does-is-mean) of what `someVar1 is { } someVar2` does.